backend/src/prompts/templates/best-tool.j2

```diff
@@ -16,6 +16,8 @@ Add if appropriate, but do not hallucinate arguments for these parameters
 
 {{ tools }}
 
+From the task you should be able to extract the parameters. If it is data driven, it should be turned into a cypher query
```

Comment:
Should we specify a specific agent for this, or can it work it out for itself?

Reply:
This is selecting the tool within the agent, so the agent has already been selected. Based on this, it can work it out itself. I don't think its 100% there but it is working for now. Before having this it wouldn't be able to select the right tool, even though there was one tool. I reckon some tweaking could improve this after adding in the semantic work as well